components/dashboard/src/components/podkit/combobox/Combobox.tsx

```diff
@@ -332,7 +333,12 @@ export const ComboboxItem: FC<ComboboxItemProps> = ({ element, isActive, classNa
     return (
         <li
             id={element.id}
-            className={cn("h-min rounded-lg flex items-center px-2 py-1.5", selectionClasses, className)}
+            className={cn(
+                "h-min rounded-lg flex items-center px-2 py-1.5",
+                selectionClasses,
+                className,
+                element?.customParentClass,
```

Comment:
Since it is hardly coupled we needed the different parent class only from certain usage of this component via their element props. So, define this new optional prop.

Reply:
See https://github.com/gitpod-io/gitpod/pull/19775/files#r1612992801